appveyor.yml

```diff
@@ -18,7 +18,7 @@ install:
   - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
   - ruby --version
   - gem --version
-  - gem install bundler
+  - gem install bundler -v 1.11.2
```

Comment:
> So should I move it to the template then?

I looked for other examples and couldn't find any (we haven't had any other RSpec builds on appveyor since bundler 1.12 was released), so my initial instinct was to wait until we saw more examples before moving it.  But you've clearly seen other examples, so yes, moving it to the template would be nice.

But that can happen outside of this PR, so I plan to merge when green.  And I don't consider it your responsibility to move it, but if you want to that would be great :).  Otherwise we'll get to it later.